HousingSearchApi/V1/Interfaces/SearchPersonsGateway.cs

```diff
@@ -0,0 +1,48 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Hackney.Core.Logging;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Boundary.Response;
+
+namespace HousingSearchApi.V1.Interfaces
+{
+    public class SearchPersonsGateway : ISearchPersonsGateway
```

Comment:
Small point: Is this naming correct given that one of the 2 methods is for getting tenures not persons?
Just noticed that may apply to other classes as well...

Reply:
Good point will rename now.